### PR TITLE
:show directive via hidden attribute

### PR DESCRIPTION
**Pull request by maennchen**
Fix for #243

### CODE REVIEW

File: lib/surface/directive/show.ex [hunk 1]
```diff
@@ -27,37 +27,52 @@ defmodule Surface.Directive.Show do
...
+              quote generated: true do
+                not unquote(value)
+              end
+        },
+        meta: show_meta
+      }
+      | non_style
+    ]
+
+    # IO.inspect attributes
```
Comment (Malian) [inline lib/surface/directive/show.ex:74]: Slight oversight 😉

Comment (maennchen) [reply lib/surface/directive/show.ex]: Fixed :)

**Comment by msaraiva** [comment]
Hey @maennchen!

Sorry for the late reply. I've been mostly on holidays for the last few days so I might be offline for a day or two.

About the PR. I like it! One thing I think we should do now is to change the name of the directive from `:show` to `:visible` as the latter sounds more intuitive as the opposite  of `hidden`. Let me know what you think and also if you want to do this in the context of this PR, otherwise I can merge it as it is and then work on the renaming on a separate PR.

Thanks a lot for you contribution.

**Comment by maennchen** [comment]
@msaraiva I actually like the name `show`. There's quite a few projects naming it the same way:

- Vue: https://vuejs.org/v2/guide/conditional.html#v-show
- Angular: https://docs.angularjs.org/api/ng/directive/ngShow
- etc.

I would therefore recommend to keep the name since it's a well understood name for people building interactive web applications.

**Comment by msaraiva** [comment]
@maennchen thanks! ❤️

**Pull request merged**